tasks/Preprocess.hpp

```diff
@@ -5,47 +5,55 @@
 
 #include "camera_base/PreprocessBase.hpp"
 #include "frame_helper/FrameHelper.h"
+#include "frame_helper/FrameHelperTypes.h"
 
 namespace camera_base {
 
-    /*! \class Preprocess 
-     * \brief The task context provides and requires services. It uses an ExecutionEngine to perform its functions.
-     * Essential interfaces are operations, data flow ports and properties. These interfaces have been defined using the oroGen specification.
-     * In order to modify the interfaces you should (re)use oroGen and rely on the associated workflow.
-     * 
+    /*! \class Preprocess
+     * \brief The task context provides and requires services. It uses an ExecutionEngine
+     to perform its functions.
+     * Essential interfaces are operations, data flow ports and properties. These
+     interfaces have been defined using the oroGen specification.
+     * In order to modify the interfaces you should (re)use oroGen and rely on the
+     associated workflow.
+     *
      * \details
      * The name of a TaskContext is primarily defined via:
      \verbatim
      deployment 'deployment_name'
          task('custom_task_name','camera_base::Preprocess')
      end
      \endverbatim
-     *  It can be dynamically adapted when the deployment is called with a prefix argument. 
+     *  It can be dynamically adapted when the deployment is called with a prefix
+     argument.
      */
-    class Preprocess : public PreprocessBase
-    {
-	friend class PreprocessBase;
+    class Preprocess : public PreprocessBase {
+        friend class PreprocessBase;
+
     protected:
-      RTT::extras::ReadOnlyPointer<base::samples::frame::Frame> oframe;	
-      frame_helper::FrameHelper frame_helper;   //helper for image processing
+        RTT::extras::ReadOnlyPointer<base::samples::frame::Frame> oframe;
+        frame_helper::FrameHelper frame_helper; // helper for image processing
 
     public:
         /** TaskContext constructor for Preprocess
-         * \param name Name of the task. This name needs to be unique to make it identifiable via nameservices.
-         * \param initial_state The initial TaskState of the TaskContext. Default is Stopped state.
+         * \param name Name of the task. This name needs to be unique to make it
+         * identifiable via nameservices. \param initial_state The initial TaskState of
```

Comment:
I assume these changes are by some auto-formatter. Here it was definitely over-aggressive. `\param` should stay at the start of the line for readability